bin/collector

```diff
@@ -1,6 +1,2 @@
 #!/usr/bin/env bash
-
-DEFAULT_TYPE=dev
-SERVICE_TYPE=${1:-$DEFAULT_TYPE}
-
-bin/sbt 'project zipkin-collector-service' "run -f zipkin-collector-service/config/collector-${SERVICE_TYPE}.scala"
+SERVICE_TYPE=${1:-dev} ./gradlew :zipkin-collector-service:run
```

Comment:
Good point. Will use gradle.properties to default and -P to override.
Nullification concern from bash is still there, but at least the change is
more idiomatic. Thanks!